Libs/MRML/Widgets/qMRMLLayoutViewFactory.cxx

```diff
@@ -407,23 +406,13 @@ void qMRMLLayoutViewFactory::onViewNodeAdded(vtkMRMLAbstractViewNode* node)
     return;
     }
 
-  ctkVTKAbstractView* view = nullptr;
-  qMRMLThreeDWidget* threeDWidget = qobject_cast<qMRMLThreeDWidget*>(viewWidget);
-  if (threeDWidget)
-    {
-    view = threeDWidget->threeDView();
-    }
-  qMRMLSliceWidget* sliceWidget = qobject_cast<qMRMLSliceWidget*>(viewWidget);
-  if (sliceWidget)
-    {
-    view = sliceWidget->sliceView();
-    }
-
-  if (view)
+  qMRMLAbstractViewWidget* abstractView = qobject_cast<qMRMLAbstractViewWidget*>(viewWidget);
```

Comment:
it is not a view but a widget, so a name like "viewWidget" would be better

Reply:
"viewWidget" is already in use for the QWidget* variable. How about "abstractViewWidget"?

Reply:
or maybe rename the `viewWidget` above to `widget` (as it is more generic)

Reply:
Ok, I've renamed it "viewWidget"->"widget", and "abstractView"->"viewWidget".